CHANGELOG.md

```diff
@@ -5,13 +5,22 @@
 **Breaking Changes**:
 
 - The `sentry_options_set_logger` function now accepts a `userdata` parameter.
-- The `name` parameter of `sentry_options_add_attachment(w)` was removed, it will
-  now be inferred from the filename of `path`.
+- The `name` parameter of `sentry_options_add_attachment(w)` was removed, it will now be inferred from the filename of `path`.
 
 **Features**:
 
+- Release Health (Sessions) is now stable, and automatic session handling is enabled by default. Use `sentry_options_set_auto_session_tracking` to change the behavior.
 - Breakpad support for Windows. This allows you to use `sentry-native` even on Windows XP! ([#278](https://github.com/getsentry/sentry-native/pull/278))
 - Add an in-process backend for Windows. As opposed to Breakpad, stack traces are generated on the device and sent to Sentry for symbolication. ([#287](https://github.com/getsentry/sentry-native/pull/287))
+- Support for the crashpad backend was fixed and enabled for Linux. ([#320](https://github.com/getsentry/sentry-native/pull/320))
+- A new `SENTRY_BREAKPAD_SYSTEM` CMake option was added to link to the system-installed breakpad client instead of building it as part of sentry.
+
+**Fixes**:
+
+- Reworked thread synchronization code and logic in `sentry_shutdown`, avoiding memory unsafety in case of an unclean shutdown. ([#323](https://github.com/getsentry/sentry-native/pull/323))
```

Comment:
```suggestion
- Reworked thread synchronization code and logic in `sentry_shutdown`, avoiding an abort in case of an unclean shutdown. ([#323](https://github.com/getsentry/sentry-native/pull/323))
```